### ISSUE TITLE
About Arena 

### ISSUE THREAD

**Issue by huskyth** [suragnair/alpha-zero-general#321]
I have a question?If I reset all maps in MCTS object if will I get the same result when I invoke the function named playGame because the value if all maps in MCTS will be common every time execution?

**Comment by aiaxmaior** [comment]
Yes — with per-game MCTS reset and temp=0 players, every game per color is identical. I've proposed a fix (seeded stochastic openings) in #<your-new-issue-number>.
